install_config/aggregate_logging.adoc

```diff
@@ -19,68 +19,87 @@ https://www.elastic.co/videos/introduction-to-the-elk-stack[ELK stack]. The EFK
 stack can be useful for viewing logs aggregated from hosts and applications,
 whether coming from multiple containers or even deleted pods.
 
-Three components make up the EFK logging stack:
+These components make up the EFK logging stack:
 
 * https://www.elastic.co/products/elasticsearch[Elasticsearch]: An object store where all logs are stored.
 * http://www.fluentd.org/architecture[Fluentd]: Gathers logs from nodes and feeds them to Elasticsearch.
 * https://www.elastic.co/guide/en/kibana/current/introduction.html[Kibana]: A web UI for Elasticsearch.
+ifdef::openshift-origin[]
+* https://www.elastic.co/guide/en/elasticsearch/client/curator/current/about.html[Curator]: Removes old logs from Elasticsearch.
+endif::openshift-origin[]
 
 Once deployed in a cluster, the stack aggregates logs from all nodes and
 projects into Elasticsearch, and provides a Kibana UI to view any logs. Cluster
 administrators can view all logs, but application developers can only view logs
-for the project the have permission to view.
+for projects they have permission to view.
 
 [NOTE]
 ====
 link:../install_config/install/prerequisites.html#managing-docker-container-logs[Managing
 Docker Container Logs] discusses the use of `json-file` logging driver options
-to manage container logs when they become full.
+to manage container logs and prevent filling node disks.
 ====
 
 == Pre-deployment Configuration
 
-Deploying the EFK stack requires:
+The deployer for the EFK stack generates the objects necessary to
+implement aggregated logging, including secrets for secure communication
+within the cluster.
 
+Before running the deployer, the following are required:
+
+* A router deployment for serving cluster-defined routes
+* Sufficient volumes for Elasticsearch cluster storage (one per replica)
+* A project, typically reserved just for this deployment
 * A *logging-deployer* secret
 * A *logging-deployer* service account with access to the secret and creation privileges
-* Sufficient volumes for Elasticsearch cluster storage
-* A router deployment for serving cluster-defined routes
 
-The deployer generates the objects necessary for communication within the
-cluster, and defines secrets and templates to implement aggregated logging.
+Perform the following steps to prepare for deployment:
 
-. Create a new project. Once implemented in a single project, the EFK stack will
-collect logs for every project within your OpenShift cluster. The examples in
-this topic use `logging` as an example project:
+. Ensure that you have deployed a router for the cluster.
+. Ensure that you have the necessary storage for Elasticsearch.
+ifdef::openshift-enterprise[]
+. Ansible-based installs should create the *_logging-deployer-template_*
+template in the *_openshift_* project.
```

Comment:
Use plain `*` for names of objects. (I don't love it, but for consistency w/ elsewhere.)